lib/ivis_opengl/gfx_api_gl.cpp

```diff
@@ -0,0 +1,87 @@
+#include "gfx_api.h"
+
+#include <GL/glew.h>
+#include <algorithm>
+#include <cmath>
+
+namespace gl_api
+{
+	auto to_gl(const gfx_api::pixel_format& format)

```

Comment:
auto return type is a c++14 feature, and we haven't enabled that yet in our build systems. Is there a strong reason to use it here?